eventdiscovery-sdk/src/main/res/values/strings.xml

```diff
@@ -26,6 +26,9 @@
     <string name="schedjoules_action_book_hotel">Hotel</string>
     <string name="schedjoules_action_book_parking">Parking</string>
     <string name="schedjoules_action_add_to_calendar">Add to calendar</string>
+
+    <string name="schedjoules_ticket_button">Get Tickets</string>
```

Comment:
Don't forget the describing comment. Also please add the context to the name, i.e. it should read like `schedjoules_event_details_ticket_button_title`. In future we may have other ticket buttons on other pages.

Note, the comment is only required in the default language.